markdown/usage/tutorials/nf_core_tutorial.md

```diff
@@ -377,8 +330,9 @@ This runs through a series of tests and reports failures, warnings and passed te
 The linting code is closely tied to the _nf-core_ template and both change over time.
 When we change something in the template, we often add a test to the linter to make sure that pipelines do not use the old method.
 
+<!-- TODO: this link does not point to the lint errors any more. I could not find the linting error docs!
```

Comment:
I couldn't find the linting error docs, are they somewhere?

Reply:
I couldn't find them either :\

Reply:
https://nf-co.re/tools/docs/latest/pipeline_lint_tests/index.html 
Okay, we should fix this as soon as possible... 😬